server/sonar-server/src/main/java/org/sonar/server/ui/JRubyFacade.java

```diff
@@ -63,6 +63,7 @@
 import org.sonar.server.plugins.UpdateCenterMatrixFactory;
 import org.sonar.server.rule.RuleRepositories;
 import org.sonar.server.user.NewUserNotifier;
+import org.sonar.server.usertoken.UserTokenAuthenticator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.server.usertoken.UserTokenAuthenticator'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)